packages/itwin/grouping-mapping-widget/src/widget/components/QueryBuilder.ts

```diff
@@ -106,9 +106,7 @@ export class QueryBuilder {
       ) !== undefined;
 
     for (let i = 0; i < propertyField.properties.length; i++) {
-      const className = propertyField.properties[
-        i
-      ].property.classInfo.name.replace(":", ".");
+      const className = propertyField.properties[i].property.classInfo.name.replace(":", ".");
```

Comment:
Why is this replacement needed here? See https://www.itwinjs.org/bis/ec/ec-name/#full-name

Reply:
Does not seem to work with `:` in this case. Generates invalid query (toast error message pops up)

Reply:
The SELECT clause probably does not like it.

Reply:
Yeah, it looks like only `FROM` and `JOIN` clauses support the `:` schema/class separator.